src/components/ToastCard/ToastCard.test.tsx

```diff
@@ -21,11 +21,9 @@ describe("Toast Card", () => {
   it("should display message", () => {
     const t = cloneDeep(toastInstanceExample);
     render(
-      <ToastCard
-        type="positive"
-        text="I am a toast message"
-        toastInstance={t}
-      />
+      <ToastCard type="positive" toastInstance={t}>
+        I am a toast message
+      </ToastCard>
```

Comment:
I've changed the toast to accept children instead of a string that gets converted to HTML.